layouts/_default/baseof.html

```diff
@@ -46,11 +50,15 @@
 			</div>
 			{{ partial "sidebar.html" . }}
 		</div>
+		{{- if .Params.partial_pre_footer }}
+			{{ partial .Params.partial_pre_footer . }}
+		{{- end }}
```

Comment:
I just saw this now when going through the changes again. What is this supposed to do?

I see why it *could* be useful to add additional content here, but I am starting to get afraid that we have too many places to add things to. That's rather confusing for the intended audience.

Reply:
Exactly. On the [web page](https://sgelbe.de/) that I used this template for, the Home Page (and only that page) features a custom visual (slider) in the area right above the footer. It's supposed to cover the whole width of the page, so putting it into normal content was not an option.

I agree that such optional and somewhat hidden feature can be daunting for users. But what alternatives are there? I failed to see any other way to tweak the HTML there, short of **copying the whole file** from the theme area to the site layouts area ... and that would not be a great solution for sure!

For reusability and amendability, having such extension points is crucial. I have no problem with them not being used in the sample website, so that can remain small and "grokable" to users. But they should be there for users who need the extension options.

I am still very new to Hugo, so please feel free to suggest other alternatives, I may just not know them! Thx.

Reply:
OK, I understand your reasoning. However, for this pull request which is very hard to track already I removed it.

However, we can discuss this in another PR. Would you like to create one? You can start with this three-liner and I will think about how to make this more general. In any case, we should document any new feature like this.